reports/report.md

```diff
@@ -957,25 +999,7 @@ The `ecrecover` function is susceptible to signature malleability. This means th
 
 
 
-## L-4: Deprecated OpenZeppelin functions should not be used
```

Comment:
Why is this being deleted??? 🤔

Reply:
OMG I've found the reason!

The ID `5463` (which is the ID of the source_unit for DeprecatedOZFunctions.sol) is duplicated! 0.8.19, 0.8.20 AND 0.8.25 all product the same ID for different ASTNodes, so they're being overwritten in the WorkspaceContext.

Holy crap @TilakMaddy , we definitely need the multiple WorkspaceContexts!

Reply:
@alexroan Is this the only thing blocking?

Reply:
@alexroan https://github.com/Cyfrin/aderyn/pull/459  <---- ICF doesn't do this. Look here !!